modern/src/MainPage.js

```diff
@@ -13,18 +19,12 @@ import CurrentLocationMap from './map/CurrentLocationMap';
 
 const useStyles = makeStyles(theme => ({
   root: {
-    height: '100%',
+    height: '100vh',
     display: 'flex',
     flexDirection: 'column',
   },
-  content: {
-    flexGrow: 1,
-    overflow: 'hidden',
-    display: 'flex',
-    flexDirection: 'row',
-    [theme.breakpoints.down('xs')]: {
-      flexDirection: 'column-reverse',
-    }
+  listContainer: {
+    zIndex: 1301,
```

Comment:
What is this for?

Reply:
Updated the structure of the code.  Used zIndex to show the device list on the top of the map.

Reply:
Can you clarify why is it not on top by default? I feel like this should be solvable without adding extra css.